1-Authentication/1-sign-in-react/SPA/package.json

```diff
@@ -3,18 +3,19 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
-        "@azure/msal-browser": "^2.37.0",
-        "@azure/msal-react": "^1.5.7",
-        "@testing-library/jest-dom": "^5.16.5",
-        "@testing-library/react": "^13.4.0",
-        "@testing-library/user-event": "^13.5.0",
-        "bootstrap": "^5.2.3",
-        "react": "^18.2.0",
-        "react-bootstrap": "^2.7.2",
-        "react-dom": "^18.2.0"
+        "@azure/msal-browser": "^3.16.0",
+        "@azure/msal-react": "^2.0.18",
+        "@testing-library/jest-dom": "^6.4.5",
+        "@testing-library/react": "^16.0.0",
+        "@testing-library/user-event": "^14.5.2",
+        "bootstrap": "^5.3.3",
+        "react": "^18.3.1",
+        "react-bootstrap": "^2.10.2",
+        "react-dom": "^18.3.1"
     },
     "devDependencies": {
-        "react-scripts": "5.0.1"
+        "@babel/plugin-transform-private-property-in-object": "^7.24.7",
```

Comment:
Are we adding a new dependency here? If yes, why?

Reply:
This is to avoid a dependency error that comes from Create React Script being ancient and not maintained.